include/zephyr/bluetooth/gatt.h

```diff
@@ -2117,7 +2103,7 @@ typedef void (*bt_gatt_subscribe_func_t)(struct bt_conn *conn, uint8_t err,
 					 struct bt_gatt_subscribe_params *params);
 
 /** Subscription flags */
-enum {
+enum bt_gatt_subs_flag {
```

Comment:
```suggestion
enum bt_gatt_sub_flag {
```

I think `sub` is a more common abbreviation for `subscription`, where `subs` is usually used to talk about `subscription**s**`